### DISCUSSION TITLE
Can the spam keywords area be unrestricted?

### DISCUSSION THREAD

**Discussion by palomamtnez** [solspace/craft-freeform#1305] [Feature Requests]
My clients have constantly come to me asking me if I could add/update spam keywords for them. It would be more beneficial and easier for them if they could change these themselves without this area being restricted in the production environment.

**Comment by kjmartens** [comment]
Thanks for your feedback on this @palomamtnez.

We plan to make some changes in the next few weeks so this can be resolved. 🙂 

It's a bit more complicated because currently those values are stored in Project Config, which is an "all or nothing" thing when using the `allowAdminChanges` config setting, but we will likely move this into the database instead.

**Comment by kjmartens** [comment]
This has been addressed in Freeform 5.5 🙂 
https://docs.solspace.com/craft/freeform/v5/setup/changelog#550
